src/doc/src/reference/manifest.md

```diff
@@ -364,7 +364,8 @@ script guide.
 ```toml
 [package]
 # ...
-links = "foo"
+# this package links with a native library called "git2", e.g. "libgit2.a" on Linux
+links = "git2"
 ```
```

Comment:
Nit: I think I'd prefer to instead say
```
    For example, to link against the native library `git2` (e.g. `libgit2.a` on Linux):

    ```toml
    ```
```
- It makes the example more directly related to what an end-user will do
- It doesn't unintentionally reinforce to users to repeat what they are commenting about in comments

Reply:
Putting the note outside of the TOML sounds good (also for consistency with the rest of the examples).  

I would caution on the exact wording, as saying "to link against…" might be misleading since adding `links` doesn't actually cause a library to be linked. I would lean more towards the proposed text ("this package links with a…"), or maybe a slight modification like "`links` indicates that the package links with a …".